api/src/services/application-exporter.service.ts

```diff
@@ -718,6 +719,38 @@ export class ApplicationExporterService {
     return res;
   }
 
+  async sortPreferencesByOrdinal(
```

Comment:
Nit but do you mind adding commenting like

```
/**
   *
   * @param
   * @param
   * @returns
   */ 
```
  to mirror the other functions in the file